### ISSUE TITLE
2025-01-31, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#68]
# Today's Arxiv Papers

(1) [DeltaLLM: Compress LLMs with Low-Rank Deltas between Shared Weights](http://arxiv.org/abs/2501.18596v1), 2025-01-30

(2) [Foundational Models for 3D Point Clouds: A Survey and Outlook](http://arxiv.org/abs/2501.18594v1), 2025-01-30

(3) [Thoughts Are All Over the Place: On the Underthinking of o1-Like LLMs](http://arxiv.org/abs/2501.18585v1), 2025-01-30

(4) [Token-Hungry, Yet Precise: DeepSeek R1 Highlights the Need for Multi-Step Reasoning Over Speed in MATH](http://arxiv.org/abs/2501.18576v1), 2025-01-30

(5) [BounTCHA: A CAPTCHA Utilizing Boundary Identification in AI-extended Videos](http://arxiv.org/abs/2501.18565v1), 2025-01-30

(6) [Semantic Web and Creative AI -- A Technical Report from ISWS 2023](http://arxiv.org/abs/2501.18542v1), 2025-01-30

(7) [Can we Retrieve Everything All at Once? ARM: An Alignment-Oriented LLM-based Retrieval Method](http://arxiv.org/abs/2501.18539v1), 2025-01-30

(8) [Illusions of Relevance: Using Content Injection Attacks to Deceive Retrievers, Rerankers, and LLM Judges](http://arxiv.org/abs/2501.18536v1), 2025-01-30

(9) [Differentially Private Steering for Large Language Model Alignment](http://arxiv.org/abs/2501.18532v1), 2025-01-30

(10) [Learn from the Past: Language-conditioned Object Rearrangement with Large Language Models](http://arxiv.org/abs/2501.18516v1), 2025-01-30

(11) [Streaming DiLoCo with overlapping communication: Towards a Distributed Free Lunch](http://arxiv.org/abs/2501.18512v1), 2025-01-30

(12) [WILDCHAT-50M: A Deep Dive Into the Role of Synthetic Data in Post-Training](http://arxiv.org/abs/2501.18511v1), 2025-01-30

(13) [CLEAR: Cue Learning using Evolution for Accurate Recognition Applied to Sustainability Data Extraction](http://arxiv.org/abs/2501.18504v1), 2025-01-30

(14) [GuardReasoner: Towards Reasoning-based LLM Safeguards](http://arxiv.org/abs/2501.18492v1), 2025-01-30

(15) [A Tool for In-depth Analysis of Code Execution Reasoning of Large Language Models](http://arxiv.org/abs/2501.18482v1), 2025-01-30

(16) [CLoQ: Enhancing Fine-Tuning of Quantized LLMs via Calibrated LoRA Initialization](http://arxiv.org/abs/2501.18475v1), 2025-01-30

(17) [ExeCoder: Empowering Large Language Models with Executability Representation for Code Translation](http://arxiv.org/abs/2501.18460v1), 2025-01-30

(18) [CALM: Unleashing the Cross-Lingual Self-Aligning Ability of Language Model Question Answering](http://arxiv.org/abs/2501.18457v1), 2025-01-30

(19) [o3-mini vs DeepSeek-R1: Which One is Safer?](http://arxiv.org/abs/2501.18438v1), 2025-01-30

(20) [GENIE: Generative Note Information Extraction model for structuring EHR data](http://arxiv.org/abs/2501.18435v1), 2025-01-30

(21) [Exploring Potential Prompt Injection Attacks in Federated Military LLMs and Their Mitigation](http://arxiv.org/abs/2501.18416v1), 2025-01-30

(22) [RbFT: Robust Fine-tuning for Retrieval-Augmented Generation against Retrieval Defects](http://arxiv.org/abs/2501.18365v1), 2025-01-30

(23) [State Stream Transformer (SST) : Emergent Metacognitive Behaviours Through Latent State Persistence](http://arxiv.org/abs/2501.18356v1), 2025-01-30

(24) [Leveraging LLM Agents for Automated Optimization Modeling for SASP Problems: A Graph-RAG based Approach](http://arxiv.org/abs/2501.18320v1), 2025-01-30

(25) [Efficient Neural Theorem Proving via Fine-grained Proof Structure Analysis](http://arxiv.org/abs/2501.18310v1), 2025-01-30

(26) [Mining for Species, Locations, Habitats, and Ecosystems from Scientific Papers in Invasion Biology: A Large-Scale Exploratory Study with Large Language Models](http://arxiv.org/abs/2501.18287v1), 2025-01-30

(27) [Leveraging Sparsity for Sample-Efficient Preference Learning: A Theoretical Perspective](http://arxiv.org/abs/2501.18282v1), 2025-01-30

(28) [Jailbreaking LLMs' Safeguard with Universal Magic Words for Text Embedding Models](http://arxiv.org/abs/2501.18280v1), 2025-01-30

(29) [Collecting Cost-Effective, High-Quality Truthfulness Assessments with LLM Summarized Evidence](http://arxiv.org/abs/2501.18265v1), 2025-01-30

(30) [Statistical multi-metric evaluation and visualization of LLM system predictive performance](http://arxiv.org/abs/2501.18243v1), 2025-01-30

(31) [Contextually Structured Token Dependency Encoding for Large Language Models](http://arxiv.org/abs/2501.18205v1), 2025-01-30

(32) [Economic Rationality under Specialization: Evidence of Decision Bias in AI Agents](http://arxiv.org/abs/2501.18190v1), 2025-01-30

(33) [Investigating Tax Evasion Emergence Using Dual Large Language Model and Deep Reinforcement Learning Powered Agent-based Simulation](http://arxiv.org/abs/2501.18177v1), 2025-01-30

(34) [RepoAudit: An Autonomous LLM-Agent for Repository-Level Code Auditing](http://arxiv.org/abs/2501.18160v1), 2025-01-30

(35) [Large Language Models for Cryptocurrency Transaction Analysis: A Bitcoin Case Study](http://arxiv.org/abs/2501.18158v1), 2025-01-30

(36) [Mixed-Precision Graph Neural Quantization for Low Bit Large Language Models](http://arxiv.org/abs/2501.18154v1), 2025-01-30

(37) [Unraveling the Capabilities of Language Models in News Summarization](http://arxiv.org/abs/2501.18128v1), 2025-01-30

(38) [Battery State of Health Estimation Using LLM Framework](http://arxiv.org/abs/2501.18123v1), 2025-01-30

(39) [Self-supervised Quantized Representation for Seamlessly Integrating Knowledge Graphs with Large Language Models](http://arxiv.org/abs/2501.18119v1), 2025-01-30

(40) [Scaling Inference-Efficient Language Models](http://arxiv.org/abs/2501.18107v1), 2025-01-30

(41) [Beyond Turn-taking: Introducing Text-based Overlap into Human-LLM Interactions](http://arxiv.org/abs/2501.18103v1), 2025-01-30

(42) [Panacea: Mitigating Harmful Fine-tuning for Large Language Models via Post-fine-tuning Perturbation](http://arxiv.org/abs/2501.18100v1), 2025-01-30

(43) [Learning to Plan & Reason for Evaluation with Thinking-LLM-as-a-Judge](http://arxiv.org/abs/2501.18099v1), 2025-01-30

(44) [LLMs can see and hear without any training](http://arxiv.org/abs/2501.18096v1), 2025-01-30

(45) [AlphaAdam:Asynchronous Masked Optimization with Dynamic Alpha for Selective Updates](http://arxiv.org/abs/2501.18094v1), 2025-01-30

(46) [Normative Evaluation of Large Language Models with Everyday Moral Dilemmas](http://arxiv.org/abs/2501.18081v1), 2025-01-30

(47) [FinanceQA: A Benchmark for Evaluating Financial Analysis Capabilities of Large Language Models](http://arxiv.org/abs/2501.18062v1), 2025-01-30

(48) [RL-based Query Rewriting with Distilled LLM for online E-Commerce Systems](http://arxiv.org/abs/2501.18056v1), 2025-01-29

(49) [A Proximal Operator for Inducing 2:4-Sparsity](http://arxiv.org/abs/2501.18015v1), 2025-01-29

(50) [Large Language Models Think Too Fast To Explore Effectively](http://arxiv.org/abs/2501.18009v1), 2025-01-29

(51) [Fault Localization via Fine-tuning Large Language Models with Mutation Generated Stack Traces](http://arxiv.org/abs/2501.18005v1), 2025-01-29

(52) [InnerThoughts: Disentangling Representations and Predictions in Large Language Models](http://arxiv.org/abs/2501.17994v1), 2025-01-29

(53) [Can Generative LLMs Create Query Variants for Test Collections? An Exploratory Study](http://dx.doi.org/10.1145/3539618.3591960), 2025-01-29

(54) [Think Smarter not Harder: Adaptive Reasoning with Inference Aware Optimization](http://arxiv.org/abs/2501.17974v1), 2025-01-29

(55) [LLMs can be Fooled into Labelling a Document as Relevant (best café near me; this paper is perfectly relevant)](http://dx.doi.org/10.1145/3673791.369843), 2025-01-29

(56) ["I Would Never Trust Anything Western": Kumu (Educator) Perspectives on Use of LLMs for Culturally Revitalizing CS Education in Hawaiian Schools](http://arxiv.org/abs/2501.17942v1), 2025-01-29

(57) [Dialogue is Better Than Monologue: Instructing Medical LLMs via Strategical Conversations](http://arxiv.org/abs/2501.17860v1), 2025-01-29

(58) [Improving Your Model Ranking on Chatbot Arena by Vote Rigging](http://arxiv.org/abs/2501.17858v1), 2025-01-29

(59) [Learning Beyond the Surface: How Far Can Continual Pre-Training with LoRA Enhance LLMs' Domain-Specific Insight Learning?](http://arxiv.org/abs/2501.17840v1), 2025-01-29

(60) [LEKA:LLM-Enhanced Knowledge Augmentation](http://arxiv.org/abs/2501.17802v1), 2025-01-29

(61) [Leveraging Multimodal LLM for Inspirational User Interface Search](http://dx.doi.org/10.1145/3706598.3714213), 2025-01-29

(62) [BreezyVoice: Adapting TTS for Taiwanese Mandarin with Enhanced Polyphone Disambiguation -- Challenges and Insights](http://arxiv.org/abs/2501.17790v1), 2025-01-29

(63) [Reasoning Over the Glyphs: Evaluation of LLM's Decipherment of Rare Scripts](http://arxiv.org/abs/2501.17785v1), 2025-01-29

(64) [AdditiveLLM: Large Language Models Predict Defects in Additive Manufacturing](http://arxiv.org/abs/2501.17784v1), 2025-01-29

(65) [2SSP: A Two-Stage Framework for Structured Pruning of LLMs](http://arxiv.org/abs/2501.17771v1), 2025-01-29

(66) [Hybrid Graphs for Table-and-Text based Question Answering using LLMs](http://arxiv.org/abs/2501.17767v1), 2025-01-29

(67) [On the Partitioning of GPU Power among Multi-Instances](http://arxiv.org/abs/2501.17752v1), 2025-01-29

(68) [Early External Safety Testing of OpenAI's o3-mini: Insights from the Pre-Deployment Evaluation](http://arxiv.org/abs/2501.17749v1), 2025-01-29

(69) [Using Code Generation to Solve Open Instances of Combinatorial Design Problems](http://arxiv.org/abs/2501.17725v1), 2025-01-29

(70) [RICoTA: Red-teaming of In-the-wild Conversation with Test Attempts](http://arxiv.org/abs/2501.17715v1), 2025-01-29

(71) [DReSS: Data-driven Regularized Structured Streamlining for Large Language Models](http://arxiv.org/abs/2501.17905v1), 2025-01-29

(72) [Planning with Vision-Language Models and a Use Case in Robot-Assisted Teaching](http://arxiv.org/abs/2501.17665v1), 2025-01-29

(73) [In-Context Meta LoRA Generation](http://arxiv.org/abs/2501.17635v2), 2025-01-29

(74) [The Imitation Game According To Turing](http://arxiv.org/abs/2501.17629v1), 2025-01-29

(75) [Uncertainty Quantification and Decomposition for LLM-based Recommendation](http://dx.doi.org/10.1145/3696410.3714601), 2025-01-29

(76) [Structured Context Recomposition for Large Language Models Using Probabilistic Layer Realignment](http://arxiv.org/abs/2501.17617v1), 2025-01-29

(77) [Semantic Consistency Regularization with Large Language Models for Semi-supervised Sentiment Analysis](http://arxiv.org/abs/2501.17598v1), 2025-01-29

(78) [GLLM: Self-Corrective G-Code Generation using Large Language Models with User Feedback](http://arxiv.org/abs/2501.17584v1), 2025-01-29

(79) [CSEval: Towards Automated, Multi-Dimensional, and Reference-Free Counterspeech Evaluation using Auto-Calibrated LLMs](http://arxiv.org/abs/2501.17581v1), 2025-01-29

(80) [Query-Aware Learnable Graph Pooling Tokens as Prompt for Large Language Models](http://arxiv.org/abs/2501.17549v1), 2025-01-29

(81) [Is Conversational XAI All You Need? Human-AI Decision Making With a Conversational XAI Assistant](http://dx.doi.org/10.1145/3708359.3712133), 2025-01-29

(82) [Towards Supporting Penetration Testing Education with Large Language Models: an Evaluation and Comparison](http://arxiv.org/abs/2501.17539v1), 2025-01-29

(83) [LLM Assistance for Pediatric Depression](http://arxiv.org/abs/2501.17510v1), 2025-01-29

(84) [Neural Spelling: A Spell-Based BCI System for Language Neural Decoding](http://arxiv.org/abs/2501.17489v1), 2025-01-29

(85) [DFPE: A Diverse Fingerprint Ensemble for Enhancing LLM Performance](http://arxiv.org/abs/2501.17479v1), 2025-01-29

(86) [AugmenTest: Enhancing Tests with LLM-Driven Oracles](http://arxiv.org/abs/2501.17461v1), 2025-01-29

(87) [Large Language Models for Single-Step and Multi-Step Flight Trajectory Prediction](http://arxiv.org/abs/2501.17459v1), 2025-01-29

(88) [Virus: Harmful Fine-tuning Attack for Large Language Models Bypassing Guardrail Moderation](http://arxiv.org/abs/2501.17433v1), 2025-01-29

(89) [Actions Speak Louder than Words: Agent Decisions Reveal Implicit Biases in Language Models](http://arxiv.org/abs/2501.17420v1), 2025-01-29

(90) [General Scene Adaptation for Vision-and-Language Navigation](http://arxiv.org/abs/2501.17403v1), 2025-01-29

(91) [MultiChallenge: A Realistic Multi-Turn Conversation Evaluation Benchmark Challenging to Frontier LLMs](http://arxiv.org/abs/2501.17399v1), 2025-01-29

(92) [Learning Free Token Reduction for Multi-Modal LLM](http://arxiv.org/abs/2501.17391v1), 2025-01-29

(93) [Context-Aware Semantic Recomposition Mechanism for Large Language Models](http://arxiv.org/abs/2501.17386v1), 2025-01-29
